### PR TITLE
Fix clustered management integration tests issues on Windows

### PR DESCRIPTION
**Pull request by anthonydahanne**
That should contribute to fixing #1436

### CODE REVIEW

File: clustered/integration-test/build.gradle [hunk 1]
```diff
@@ -52,7 +52,7 @@ test {
   environment 'JAVA_OPTS', '-Dcom.tc.l2.lockmanager.greedy.locks.enabled=false'
   //If this directory does not exist, tests will fail with a cryptic assert failure
   systemProperty 'kitInstallationPath', "$unzipKit.destinationDir/${project(':clustered:clustered-dist').archivesBaseName}-$project.version-kit"
-  systemProperty 'managementPlugins', ["management-model", "management-registry", "sequence-generator", "management-entity-server", "monitoring-service", "monitoring-service-entity"].collect { String artifact -> project.configurations.testCompile.find { it.name.startsWith("$artifact-$parent.managementVersion") } }.join(':')
+  systemProperty 'managementPlugins', ["management-model", "management-registry", "sequence-generator", "management-entity-server", "monitoring-service", "monitoring-service-entity"].collect { String artifact -> project.configurations.testCompile.find { it.name.startsWith("$artifact-$parent.managementVersion") } }.join(File.pathSeparator)
```
Comment (henri-tremblay) [inline clustered/integration-test/build.gradle]: Will be cleaner to use `File.pathSeparatorChar`

**Comment by henri-tremblay** [comment]
EhcacheManagerToStringTest seems to have more issues. All hardcoded path are using slashed (it is working on Windows just because Windows is nice with slashes)

File: clustered/integration-test/build.gradle (hunk 1 above)
Comment (anthonydahanne) [inline clustered/integration-test/build.gradle]: tried that and gradle failed with : 
```
Build file 'Z:\ehcache\ehcache3\clustered\integration-test\build.gradle' line: 55

* What went wrong:
A problem occurred evaluating project ':clustered:integration-test'.
> No signature of method: java.util.ArrayList.join() is applicable for argument types: (java.lang.Character) values: [;]
Possible solutions: join(java.lang.String), min(), min(java.util.Comparator), min(groovy.lang.Closure), wait(), pop()
```

Comment (henri-tremblay) [inline clustered/integration-test/build.gradle]: My bad. Groovy is converting ':' to String.

**Comment by henri-tremblay** [comment]
LGTM. Waiting for checks to merge

File: clustered/integration-test/build.gradle (hunk 1 above)
Comment (mathieucarbou) [inline clustered/integration-test/build.gradle]: Note: The goal here is only to put any "special" char to be able to split again de long line. So any char would work as long as none as folders with special chars, We could have put space also

**Pull request merged**